contributed_definitions/NXcalibration.nxdl.xml

```diff
@@ -99,6 +121,34 @@
         <doc>
              Vector containing the data coordinates in the original uncalibrated axis
         </doc>
+        <attribute name="symbol">
+            <doc>
+                 The symbol of the axis to be used in the fit_function, e.g., `energy`, `E`.
+                 This should comply to python variable naming conventions.
+            </doc>
+        </attribute>
+        <attribute name="input_path">
+            <doc>
+                 The path from which this data is derived, e.g., raw detector axis.
+                 Should be a valid NeXus path name, e.g., /entry/instrument/detector/raw.
+            </doc>
+        </attribute>
+        <dimensions rank="1">
+            <dim index="1" value="ncal"/>
+        </dimensions>
+    </field>
+    <field name="input_SYMBOL" type="NX_FLOAT" units="NX_ANY">
+        <doc>
+             Additional input axis to be used in the formula.
+             The part after `input_` is used as the symbol to be used in the `fit_function`, i.e.,
+             if the field name is `input_my_field` you should refer to this axis by `my_field` in the `fit_function`.
```

Comment:
It would be very good to have an example for how to use this somewhere.

Reply:
Yes, I'll try to build one with a spin resolved measurement

Reply:
Example will be added in #77